services/workflows-service/scripts/generate-end-user.ts

```diff
@@ -132,6 +132,7 @@ export const generateBusiness = ({
         workflowDefinitionId,
         createdAt: faker.date.recent(2),
         projectId: projectId,
+        tags: ['manual_review'],
```

Comment:
Should get the tag configured to the initial state, if not we should still use the tags enum here.